tests/unit/amo/components/TestFeaturedAddons.js

```diff
@@ -20,25 +21,23 @@ describe('<FeaturedAddons />', () => {
   });
 
   function render({ ...props }) {
-    return shallowRender(
-      <FeaturedAddonsBase i18n={getFakeI18nInst()} {...props} />
-    );
+    return shallow(<FeaturedAddonsBase i18n={getFakeI18nInst()} {...props} />);
   }
 
   it('renders a FeaturedAddons page with no add-ons set', () => {
     store.dispatch(
       featuredActions.getFeatured({ addonType: ADDON_TYPE_EXTENSION }));
     const root = render(mapStateToProps(store.getState()));
 
-    expect(root.props.children[0].props.children).toContain('More Featured Extensions');
+    expect(root.childAt(0).contains('More Featured Extensions')).toBeTruthy();
```

Comment:
This should be [`expect(root.childAt(0)).toIncludeText('More Featured Extensions')`](https://github.com/blainekasten/enzyme-matchers#toincludetexttextstring).